waku/v2/rpc/waku_rpc.go

```diff
@@ -28,7 +32,13 @@ func NewWakuRpc(node *node.WakuNode, address string, port int) *WakuRpc {
 		log.Error(err)
 	}
 
-	err = s.RegisterService(&RelayService{node}, "Relay")
+	relayService := &RelayService{
```

Comment:
What do you think of creating a `NewRelayService` function that creates a `*RelayService`, so `WakuRpc` does not have to be aware of the internals of `RelayService` struct

Reply:
Agree, added